tools/gh_lists.py

```diff
@@ -55,22 +56,36 @@ def backtick_repl(matchobj):
 
         for issue in items:
             msg = "* `#{0} <{1}>`__: {2}"
-            # sanitize whitespace, `, and *
+            # sanitize whitespace
             title = re.sub("\\s+", " ", issue.title.strip())
-            title = re.sub("([^`]|^)`([^`]|$)", "\g<1>``\g<2>", title)
+
+            # substitute any single backtick not adjacent to a backtick
+            # for a double backtick
+            title = re.sub("(?P<pre>(?:^|(?<=[^`])))`(?P<post>(?=[^`]|$))",
```

Comment:
This means:

- group "pre", which does not consume input, matches either the start of the title or any character which isn't a backtick.
- ` matches a backtick.
- group "post", which does not consume input, matches either any character which isn't a backtick, or the end of the title.

Hence the whole pattern matches any backtick which isn't immediately preceded or followed by another backtick.